installer/templates/phx_ecto/data_case.ex

```diff
@@ -0,0 +1,65 @@
+defmodule <%= application_module %>.DataCase do
```

Comment:
I am not sure about DataCase but I am 100% sure DatabaseCase is not what we
want. I would wait for the HTML generator to be ready before making a
decision in here. One option is to simply call it MyApp.Case, as it will be
shared by contexts and schemas equally (most likely).

On Tuesday, August 16, 2016, Michał Muskała <notifications@github.com>
wrote:

> In installer/templates/phx_ecto/data_case.ex
> <https://github.com/phoenixframework/phoenix/pull/1871#discussion_r74918414>
> :
>
> > @@ -0,0 +1,65 @@
> > +defmodule <%= application_module %>.DataCase do
>
> Should this be a DataCase or rather DatabaseCase - it's closely related
> to ecto, so I don't think we should try making it abstract to general
> concept of data. There might be many sources of data in an application and
> not all should use this case template.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/phoenixframework/phoenix/pull/1871/files/17f4a98684db9672d2d9474048c8b3f83d088751#r74918414>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAAlbkfNG6ZhmENdZoCY3XT403NUJbd0ks5qgaEqgaJpZM4JklzI>
> .
>


-- 


*José Valimwww.plataformatec.com.br
<http://www.plataformatec.com.br/>Founder and Director of R&D*